R/scale.R

```diff
@@ -110,6 +110,20 @@ prep.step_scale <- function(x, training, info = NULL, ...) {
 
   sds <- sds * x$factor
 
+  which_sd <- which(sds ==0)
+  if (length(which_sd) > 0) {
+    with_na <- names(which_sd)
+    with_na_str <- paste(paste0("`", with_na, "`"), collapse = ", ")
+    rlang::abort(c(
+      "Some columns have Zero variance",
+      "i" = 'Consider running step_zv to remove those columns.',
```

Comment:
```suggestion
      "Some columns have zero variance so normalization is not defined:",
      glue_cols,
      "i" = 'Consider `step_zv()` to remove those columns.',
```
Put the column names before the instruction about `step_zv()`.